packages/deployments/contracts/src/cli/init/config/mainnet/production.ts

```diff
@@ -397,6 +414,7 @@ export const MAINNET_PRODUCTION_INIT_CONFIG: InitConfig = {
       "1634886255": "0x5C711DB90dEc0a5B81C626968DEa4187a7f9C1F2",
       "6450786": "0x9435Ba7C661a0Fd477deED640491de8c100325A7",
       "6778479": "0x7616Bc6d0dee5E250BA5b3dDa6cbbB71786FB638",
+      "1818848877": "0xade09131C6f43fe22C2CbABb759636C43cFc181e",
     },
     watchers: {
       allowlist: ["0xade09131C6f43fe22C2CbABb759636C43cFc181e"],
```

Comment:
we are missing addresses of our mainnet deployed watchers here, need to add them

Reply:
Updated

Reply:
> we are missing addresses of our mainnet deployed watchers here, need to add them

What other new watchers do we have, the V2 watchers ? @LayneHaber

Reply:
added ours, requested info from others. we can add with protocol admin, we should wait to whitelist until they are confirmed updated with the latest version